xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1427,6 +1371,10 @@ CAMLCodec::CAMLCodec() : CThread("CAMLCodec")
   am_private = new am_private_t;
   memset(am_private, 0, sizeof(am_private_t));
   m_dll = new DllLibAmCodec;
+  if(!(m_dll->CanLoad() && m_dll->Load())) {
```

Comment:
> CanLoad doesn't work on Android. Does it on Linux?

We could just leave it out, since `DllDynamic::Load()` seems to return a boolean value to indicate if loading was successful. I think we could simply use that.

> Also please follow formatting guidelines
> - 2 spaces indent (for info, haven't seen an issue)
> - Opening bracket on own line

I tried to match the formatting of the rest of the code in that file, but I [took the wrong line as an example](https://github.com/koying/xbmc/pull/2/files#diff-95dab1a34c0c1632e59cccd0becf7d79R1229).  ;-)

I fixed the issues and amended the commit to keep the history clean. I'll recompile and test, but since i'm compiling on the device itself, that will take a while.